metaflow/cli.py

```diff
@@ -1066,6 +1068,10 @@ def start(
     if decospecs:
         decorators._attach_decorators(ctx.obj.flow, decospecs)
 
+    # We create an instance of SystemMonitor and SystemLogger respectively
```

Comment:
nit, I would move this up to where ctx.obj.monitor and event_logger are created.